xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.cpp

```diff
@@ -661,33 +681,48 @@ void CActiveAESink::Process()
   }
 }
 
-void CActiveAESink::EnumerateSinkList(bool force)
+void CActiveAESink::EnumerateSinkList(bool force, std::string driver)
 {
   if (!m_sinkInfoList.empty() && !force)
     return;
 
   if (!CAESinkFactory::HasSinks())
     return;
 
+  std::vector<AE::AESinkInfo> tmpList(m_sinkInfoList);
+
   unsigned int c_retry = 4;
   m_sinkInfoList.clear();
-  CAESinkFactory::EnumerateEx(m_sinkInfoList, false);
+
+  if (!driver.empty())
+  {
+    for (auto const &info : tmpList)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types